src/app/mainview/mainview.component.ts

```diff
@@ -174,6 +180,13 @@ export class MainviewComponent implements OnInit, OnDestroy {
             if (region) { this.showRegion = true; }
             window.scrollTo(0, 0);
         });
+
+
+        this._nssService.currentItem.subscribe(item => this.item = item);
+        this._nssService.currentStatisticGroup.subscribe(statisticGroup => this.statisticGroup = statisticGroup);
+        this._nssService.currentRegRegion.subscribe(regRegion => this.regRegion = regRegion);
+        this._nssService.currentClone.subscribe(clone => this.clone = clone);
```

Comment:
Same as above, I don't think these variables are actually used anywhere else in the mainview component.